### PR TITLE
Update "Get started with Progressive Web Apps"

### PR DESCRIPTION
**Pull request by vchapel**
Fixed link per work item 36011249 and issue 1351

### CODE REVIEW

File: microsoft-edge/progressive-web-apps-chromium/get-started.md [hunk 1]
```diff
@@ -115,21 +115,20 @@ Service workers are background tasks that intercept network requests from your w
...
 Service workers are defined in a special JavaScript file.  For more information, navigate to [Using Service Workers][MDNUsingServiceWorkers] and [Service Worker API][MDNServiceWorkerApi].  
 
-To build a service worker in your project, use the **Cache-first network** service worker recipe from [PWA Builder][PwaBuilderServiceWorker].  
+To build a service worker in your project, use the **Cache-first network** service worker recipe.  
 
-1.  Navigate to [pwabuilder.com/serviceworker][PwaBuilderServiceWorker], select the **Cache-first network** service worker, and select the **Download** button.  The downloaded file contains the following files:
+1. Copy the source files [pwabuilder-sw-register.js][PwaBuilderServiceWorkerRegisterJS] and [pwabuilder-sw.js][PwaBuilderServiceWorkerJS] to the `public` folder in your web app project.
     
     *   `pwabuilder-sw-register.js`  
     *   `pwabuilder-sw.js`
```
Comment (Reezaali) [inline microsoft-edge/progressive-web-apps-chromium/get-started.md:123]: Can we delete the bulleted list since the links have now been moved up?

**Review by Reezaali — CHANGES_REQUESTED** [review]
1 minor change needed. Please review.

File: microsoft-edge/progressive-web-apps-chromium/get-started.md (hunk 1 above)
Comment (vchapel) [reply microsoft-edge/progressive-web-apps-chromium/get-started.md:123]: sure

File: microsoft-edge/progressive-web-apps-chromium/get-started.md [hunk 2]
```diff
@@ -3,7 +3,7 @@ description: This guide gives you an overview of PWA basics and tools for buildi
 title: Get started with Progressive Web Apps (Chromium)
```
Comment (mikehoffms) [inline microsoft-edge/progressive-web-apps-chromium/get-started.md:3]:
```suggestion
title: Get started with Progressive Web Apps
```
Also change # h1 to match, if make this change.

**Review by jgw96 — APPROVED** [review]

**Review by Reezaali — APPROVED** [review]

**Pull request merged**